pkg/madmin/API.md

```diff
@@ -245,6 +245,18 @@ __Example__
 ### HealObject(bucket, object string, isDryRun bool) (HealResult, error)
 If object is successfully healed returns nil, otherwise returns error indicating the reason for failure. If isDryRun is true, then the object is not healed, but heal object request is validated by the server. e.g, if the object exists, if object name is valid etc.
 
+| Param  | Type  | Description  |
+|---|---|---|
+|`h.State` | _HealState_ | Represents the result of heal operation. It could be one of `NotHealed`, `PartiallyHealed` or `CompletelyHealed`. |
+
+
+| Value | Description |
+|---|---|
+|`NotHealed` | Object/Upload wasn't healed on any of the disks |
```

Comment:
@harshavardhana yes, thanks for catching that. I have updated the PR fixing this.